onnxruntime/core/graph/model.cc

```diff
@@ -500,6 +504,37 @@ static Status LoadModel(const T& file_path, std::shared_ptr<Model>& p_model,
 
 template <typename T>
 static Status SaveModel(Model& model, const T& file_path) {
+#if defined(__wasm__)
```

Comment:
- Should we only allow this feature in DEBUG build?

```c
#if defined(__wasm__) && !defined(NDEBUG)
```

Reply:
No need to put it under debug in my view.
Size wise it should not matter. Don't think there is a security impact since you can always fetch the model and get the same info from there.